src/components/LiquidityMining/Create/Steps/PreviewAndCreate/PoolSummary/index.tsx

```diff
@@ -8,7 +8,7 @@ import { DateTime } from 'luxon'
 import { unwrappedToken } from '../../../../../../utils/wrappedCurrency'
 
 interface PoolSummaryProps {
-  liquidityPair: Pair | Token | null
+  liquidityPair?: Pair | Token
```

Comment:
Can you break this into 2 props?

Reply:
I don't think we will do much if we break it down here since we still need to check what type it is...We will have extra props for no reason...

Reply:
IMO two different types should be two different props. And the prop name is `liquidityPair` but can be a token. Mixing types is like saying that we have something that could be an egg and an orange at the same time. If I use this in the future I will only pass liquidityPair to this component and no token.